### PR TITLE
Remove animations on compact change

### PR DESCRIPTION
**Pull request by WPFerg**
Closes (and uses the approach in) #516

### CODE REVIEW

File: src/main/main-directive.js [hunk 1]
```diff
@@ -2,12 +2,21 @@
...
                 controller: 'MainCtrl',
-                controllerAs: 'mainCtrl'
+                controllerAs: 'mainCtrl',
+                link: (scope, element) => {
+                    scope.$on('compactChanging', () => {
+                        element.addClass('disable-transitions');
+
+                        $timeout(() => {
+                            element.removeClass('disable-transitions');
+                        }, 100);
```
Comment (owennw) [inline src/main/main-directive.js]: I wonder if there should be two events: `before-compact-change` and `after-compact-change`?

That way you might not need the `$timeout`, and you can broadcast on either side of the compact change in the toolbar-controller. Or do async things mess that up?

Comment (WPFerg) [inline src/main/main-directive.js]: I've just tried it, and it doesn't work. Probably something to do with how angular's events are processed.

**Comment by bjedrzejewski** [comment]
@WPFerg can you update? I will test it and if it works great, will merge

**Comment by WPFerg** [comment]
@bjedrzejewski Done

**Comment by bjedrzejewski** [comment]
@WPFerg ups - travis cried

**Comment by WPFerg** [comment]
@bjedrzejewski Looks like it fell at `npm install`, I did a `grunt ci` and it passed, give it a go anyway and we'll try again if you think it's ready to merge.

**Pull request merged**